spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe 'Registrations', type: :request do
+  describe 'POST #create' do
```

Comment:
It's more common to use on request specs something like `describe 'POST /signup'` or something like that, the `'describe 'POST #create'` is more used on ~~old~~ controller specs.